src/chapters/ldap-auth.xml

```diff
@@ -384,6 +384,17 @@ dn: cn={4}guacConfigGroup,cn=schema,cn=config
                                 have no effect on Guacamole configurations.</emphasis></para>
                     </listitem>
                 </varlistentry>
+                <varlistentry>
+                    <term><property>ldap-dereference-aliases</property></term>
+                    <listitem>
+                        <para>Controls whether or not the LDAP connection follows (dereferences) aliases
+                            as it searches the tree.  There are four possible values: never (the default),
+                            finding, searching, and always.  Never means aliases will never be followed;
```

Comment:
Similar to the above, starting this sentence with "Never means ..." (uppercase N) could lead to confusion when specifying the `ldap-dereference-aliases` property, as the values are strictly interpreted. Things need to be rephrased/reorganized somehow such that the value itself can be written lowercase, exactly as it would appear in `guacamole.properties`, and in quotes so it's clear that we're referring to a strict literal value.